src/components/VaultRow/VaultRow.tsx

```diff
@@ -157,8 +208,17 @@ export const VaultRow: FC<VaultTableRowProps> = ({
         <Accordion.Content className="col-span-full overflow-hidden ui-state-closed:animate-accordion-close ui-state-open:animate-accordion-open max-lg:-mx-3">
           {/* Desktop: forms */}
           <div className="mt-6 grid grid-cols-2 gap-4 max-lg:hidden">
-            <VaultDepositForm {...props} />
-            <VaultWithdrawForm {...props} />
+            {isCompounderProduct ? (
```

Comment:
I sort of think we should accept these components (depositForm, withdrawForm, apy, userBalance, whatever else is different between the two) as props instead of doing the logic in here but I'm not really sure yet and this works fine.

Reply:
Pass these forms as parameter is pretty heavy. Let's do it as it is ATM and reiterate it later. Stabilize it first for now.

Reply:
I'm in agreement 💯